### ISSUE TITLE
Wrong Download Client name is reported for `Grabbed` events

### ISSUE THREAD

**Issue by agneevX** [Sonarr/Sonarr#4801]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I have three download clients added to Sonarr. All three of them are qBittorrent but have different common names in Sonarr.

Sonarr however shows the same name "qBittorrent" instead of using the common name assigned.

<img width="446" alt="Screen Shot 2021-12-14 at 2 45 41 PM" src="https://user-images.githubusercontent.com/19761269/145969446-f8e3f8e1-afd7-4e6d-be02-4c0f1c2db2fe.png">

In this case, the release was sent to the `oc-bom1` client

<img width="646" alt="Screen Shot 2021-12-14 at 2 46 00 PM" src="https://user-images.githubusercontent.com/19761269/145969558-b5d5480f-a3e3-46fd-9861-6226ed84327d.png">

### Expected Behavior

The user-set name should be displayed instead.

### Steps To Reproduce

1. Add a qBittorrent download client with a name other than qBittorrent
2. Grab a release
3. Go to History...

### Environment

```markdown
- OS: Ubuntu 20.04
- Sonarr: 3.0.6.1399
- Docker Install: Yes
- Using Reverse Proxy:
- Browser:
```

### What branch are you running?

Develop

### Trace Logs?

-

### Anything else?

-

**Comment by thezoggy** [comment]
guessing it should be downloadClient.name ?

**Comment by agneevX** [comment]
This is still an issue:

<img width="739" alt="Screen Shot 2022-01-01 at 1 19 00 PM" src="https://user-images.githubusercontent.com/19761269/147846427-94aebd75-78ef-4500-9e73-87abed34123b.png">
